install_config/persistent_storage/selector_label_binding.adoc

```diff
@@ -111,6 +111,58 @@ spec:
 labels of targeted PV.
 ====
 
+[[selector-label-volume-bind]]
+=== Optional: Bind the PVC to a specific PV when using a selector
+
+As the *cluster-admin* user, you can bind the PVC to a specific PV only.
+
+To reserve a PV for specific tasks, you have two options: create a specific storage class, or pre-bind the PV to your PVC.
+
+. Request a specific storage class for the PV by specifying the storage class’s name.
++
+The following resource shows the required values that you use to configure a StorageClass. This example uses the AWS ElasticBlockStore (EBS) object definition.
+
```

Comment:
The bug reporter requested a storage class spec for binding purposes, so I borrowed from this sample: https://docs.openshift.com/container-platform/4.3/storage/dynamic-provisioning.html#basic-storage-class-definition_dynamic-provisioning
Could use help if there is a better example we could use for this context.